recipes/eleuther_eval.py

```diff
@@ -187,15 +188,16 @@ def evaluate(self) -> None:
 
         task_dict = get_task_dict(self._tasks)
         logger.info(f"Running evaluation on {self._tasks} tasks.")
-        eleuther_output = evaluate(
+        output = evaluate(
             model_eval_wrapper,
             task_dict,
             limit=self._limit,
         )
 
         logger.info(f"Eval completed in {time.time() - t1:.02f} seconds.")
-        for task, res in eleuther_output["results"].items():
-            logger.info(f"{task}: {res}")
+
+        formatted_output = make_table(output)
+        print(formatted_output)
```

Comment:
Why switch from logger to print here? Don't we use log in all our other recipes instead of print?

Reply:
Logger formats it weirdly. See:

```
2024-05-15:11:42:06,560 INFO     [eleuther_eval.py:200] |    Tasks     |Version|Filter|n-shot|Metric|Value |   |Stderr|
|--------------|------:|------|-----:|------|-----:|---|-----:|
|truthfulqa_mc2|      2|none  |     0|acc   |0.8127|±  |0.1046|
```

Reply:
This shouldn't be an issue for this recipe so I'll approve, but the main issue with print is that it gets lost with distributed runs.

Reply:
When this becomes distributed, I'll fix.